feat: add image and input components

irfan-nawaz opened the pull request:
- merging image and input components.
- refactor code of other components.
- add CSS variables for box-shadows

Commit:
feat: add image and input components

- merging image and input components.
- refactor code of other components.
- add CSS variables for box-shadows

starver20 commented on Components/Input/input.css:
```diff
@@ -0,0 +1,41 @@
+.default-input input:focus{
+    border-bottom : 2px solid var(--primary-color);
+}
+
+/* Error validator for input*/
+.error-input input:focus{
+    border-bottom : 2px solid var(--error-color);
+}
+
```
Error effect color should remain all the time, not only on focus

irfan-nawaz replied:
done 🙌🏼

> At some places you have used hyphen (-) and most of the places you've used under score (_) in the naming convention. Try to use only one of these as Pranshu has mentioned earlier about it. Other than that every thing looks good. You can merge.

thanks, guys for reminding me I have done the suggested changes

starver20 reviewed (approved):
All good except the suggestion.👍🏻

shaik-saad reviewed (approved):
At some places you have used hyphen (-) and most of the places you've used under score (_) in the naming convention. Try to use only one of these as Pranshu has mentioned earlier about it. Other than that every thing looks good. You can merge.

Commit:
fix: add suggested changes

followed common CSS naming style,
and made the input error and success permanent.
